languages/csharp/reference/exercise-concepts/isbn-verifier.md

```diff
@@ -0,0 +1,19 @@
+[Example implementation](https://github.com/exercism/csharp/blob/master/exercises/isbn-verifier/Example.cs)
+
+ ## General
+ - functions: used as the main entry point to this exercise
+ - static functions: used as the main entry point to this exercise
+ - function arguements: input string is passed as an arguement
+ - return values: returning value from method
+ - visibility: test class and test methods are `public`
+ - imports: importing types with using statements
+ - namespaces: knowing where to find Regular Expression methods
+ - regex: use regex to implement the checker
+ - strings: used to store the isbn numbers
+ - for loop: iterate over the numbers to check if valid 
+ - modulus: checking return value has no remainder
+ - conditionals: used for checking values
+ - collasce: combine coniditonals to check for isbn
```

Comment:
```suggestion
 - coalesce operator: combine conditonals to check for isbn
```

Reply:
I changed this to ternary `?` operator, because we're not doing any coalesce `??`